.github/workflows/update-version.yml

```diff
@@ -0,0 +1,49 @@
+#
+# Copyright 2024 Adobe. All rights reserved.
+# This file is licensed to you under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License. You may obtain a copy
+# of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+# OF ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+#
+
+name: Update Versions
+on:
+  workflow_dispatch:
+    inputs:
+      notificationbuilder-version: 
+        description: 'New version to use for the NotificationBuilder. Example: 3.0.0'
+        required: false
```

Comment:
Can you also add a field to update core dependency and test it with version.sh?
https://github.com/adobe/aepsdk-userprofile-android/blob/main/.github/workflows/update-version.yml#L10